<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rozpoznawanie obrazów przy użyciu sztucznej sieci neuronowej</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>540</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Rozpocznij uczenie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testujButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>540</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Testuj bez uczenia</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="testDataButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="sciezkaDaneUczaceTextEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ścieżka do danych uczących:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trainingDataButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="sciezkaDaneTestoweTextEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ścieżka do katalogów z danymi testowymi:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trainingDataButton_2">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>70</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>421</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Sieć neuronowa</string>
    </property>
    <widget class="QSpinBox" name="liczbaNeuronow1SpinBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>5000</number>
     </property>
     <property name="value">
      <number>500</number>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="szerokoscSpinBox">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="liczbaNeuronow2SpinBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="wysokoscSpinBox">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>20</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Liczba neuronów ukrytej warstwy 1:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Liczba neuronów ukrytej warstwy 2:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Szerokość:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Wysokość:</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="logTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>571</width>
      <height>521</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="wyswietlWPrzegladarce">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>540</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyświetl w przeglądarce zdjęć</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prawo">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>540</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lewo">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>540</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>571</width>
      <height>521</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUstawienia">
    <property name="title">
     <string>Ustawienia</string>
    </property>
    <addaction name="actionZmie_ustawienia"/>
   </widget>
   <addaction name="menuUstawienia"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionWczytaj">
   <property name="text">
    <string>Wczytaj...</string>
   </property>
  </action>
  <action name="actionZmie_ustawienia">
   <property name="text">
    <string>Zmień ustawienia...</string>
   </property>
  </action>
  <action name="actionImformacje_o_programie">
   <property name="text">
    <string>Informacje o programie</string>
   </property>
  </action>
  <action name="actionSpecyfikacja">
   <property name="text">
    <string>Specyfikacja</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
